### ISSUE TITLE
explicit metadata

### ISSUE THREAD

**Issue by rickj** [daisy/ace-core#22]
When reporting the metadata at the top of the report, I'd love to have it notate missing required items.  I ran it on a test file, and (of course) it does not have any of the accessibility metadata to report:
```
  },
  "dct:date": "8/7/2017, 11:52:29 AM",
  "dct:title": "ACE Report",
  "dct:description": "Accessibility Checker Report",
  "earl:testSubject": {
    "url": "/Users/nbrown1/Downloads/hack-epub.epub",
    "dct:title": "Hack Education: Top Ed-Tech Trends of 2016",
    "dct:identifier": "urn:isbn:hack-education",
    "metadata": {
      "dc:title": "Hack Education: Top Ed-Tech Trends of 2016",
      "dc:creator": "Audrey Watters",
      "dc:language": "en",
      "dc:identifier": "urn:isbn:hack-education",
      "dcterms:modified": "2016-12-28T21:26:35Z"
    }

```

I would like to see it report that these items (and others?) are missing, or empty:
accessMode
accessibilityFeature
accessibilityHazard
accessibilitySummary
accessModeSufficient
accessibilityAPI
accessibilityControl
a11y:certifiedBy

**Comment by rdeltour** [comment]
I agree, thanks for the feedback!
We also envisioned trying to propose values for missing metadata (or check existing metadata) based on what we found in the content.

**Comment by marisademeglio** [comment]
added summary of accessibility metadata (incl what's missing) to the Ace report. the rest of this topic is to-be-continued: https://github.com/daisy/ace-core/issues/35
